app/views/expenses/_index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="flex flex-col items-center gap-3">
+   <h5 class="bg-sky-700/70 text-lg flex font-light text-white justify-center h-6">Please reload the page before adding transaction</h5>
+   <h2 class="font-bold text-lg text-sky-700/80">Add Transaction</h2> 
```

Comment:
- [ ] For adding new transactions reloading the page is not good for users. I know you said you will fix after approval but you have still time so please fix it.

![image](https://user-images.githubusercontent.com/38635158/167253492-69803ed5-2abf-47d8-ae80-7e25677129c6.png)